[0m22:18:37.464846 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016151841070>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000161518812B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016151881040>]}


============================== 22:18:37.471859 | ce29e93e-723a-4905-b8ad-ba886b5a6f78 ==============================
[0m22:18:37.471859 [info ] [MainThread]: Running with dbt=1.8.9
[0m22:18:37.471859 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Lenovo\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init dbt_project', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:18:37.526030 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'WindowsPath'.. Dictionary: {'dir': WindowsPath('C:/Users/Lenovo/.dbt')}
[0m22:18:37.527030 [info ] [MainThread]: Creating dbt configuration folder at 
[0m22:18:37.529033 [debug] [MainThread]: Starter project path: C:\projects\idf-task\.venv\Lib\site-packages\dbt\include\starter_project
[0m22:18:37.552105 [info ] [MainThread]: 
Your new dbt project "dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m22:18:37.553815 [info ] [MainThread]: Setting up your profile.
[0m22:18:49.188133 [info ] [MainThread]: No sample profile found for clickhouse.
[0m22:18:49.191096 [debug] [MainThread]: Command `dbt init` succeeded at 22:18:49.191096 after 11.84 seconds
[0m22:18:49.191096 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016151A101D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000161519FE5A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001615149D5B0>]}
[0m22:18:49.192140 [debug] [MainThread]: Flushing usage events
[0m20:19:56.653072 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000238C280A8D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000238C3CE3E30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000238C3CE0470>]}


============================== 20:19:56.661145 | 5ce7e262-5ff7-4b5e-b98b-9eb81236544b ==============================
[0m20:19:56.661145 [info ] [MainThread]: Running with dbt=1.8.9
[0m20:19:56.661145 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Lenovo\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt test --select people_in_space_aggregated', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m20:19:56.662108 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\projects\idf-task\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m20:19:56.663107 [debug] [MainThread]: Command `dbt test` failed at 20:19:56.663107 after 0.16 seconds
[0m20:19:56.664144 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000238C5950170>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000238C280A8D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000238C608C0E0>]}
[0m20:19:56.664144 [debug] [MainThread]: Flushing usage events
[0m23:26:33.062774 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002031C2A1250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002031B4F7E00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000203189FAA80>]}


============================== 23:26:33.066802 | cbc6f703-9923-408f-bda5-d342d3cb3940 ==============================
[0m23:26:33.066802 [info ] [MainThread]: Running with dbt=1.8.9
[0m23:26:33.067794 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Lenovo\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt test --select people_in_space_aggregated', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m23:26:33.068773 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\projects\idf-task\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m23:26:33.070774 [debug] [MainThread]: Command `dbt test` failed at 23:26:33.070774 after 0.14 seconds
[0m23:26:33.070774 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002031BB626F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002031C0002C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002031B4F7E00>]}
[0m23:26:33.070774 [debug] [MainThread]: Flushing usage events
